l10n_it_account/__openerp__.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Abstract srl (<http://www.abstract.it>)
+#    Copyright (C) 2015 Agile Business Group sagl (<http://www.agilebg.com>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Italian Localization - Account',
+    'version': '0.1',
+    'category': 'Localization/Italy',
+    'description': "Accounting-related modules helpers""",
+    'author': "Agile Business Group,Abstract,Odoo Community Association (OCA)",
+    'website': 'http://www.odoo-italia.org',
+    'license': 'AGPL-3',
+    "depends": ['base', 'account'],
+    "data": [
+        'views/account_view.xml',
+        'reports/account_reports_view.xml',
+    ],
+    "qweb": [],
+    "demo": [],
+    "test": [],
+    "active": False,
```

Comment:
This key is deprecated and it's an alias for ```auto_install``` whose default value is already ```False```. So please remove it.